config_test.go

```diff
@@ -65,6 +65,18 @@ func TestHostCredentials_Set(t *testing.T) {
 			err:    errors.New("parse host 'host1:5432=' failed: username empty"),
 			output: map[string]postgres.Credentials{},
 		},
+		{
+			name:  "host with ssl configured",
+			value: "host1:5432=user1:pass1=params=sslmode=enabled",
```

Comment:
To me, this makes no sense

Reply:
And yet you approved? 🙈 

The current format is `key=value` pairs of a host name with port and a username with password.

```
    host   |    user      |                 params
host1:5432 = user1:pass1
host1:5432 = user1:pass1  =   sslmode=strict&connection_timeout=5s
```

The last key=value pair is `params=<some PostgreSQL connection params>`.

When writing this, it becomes clear that `params=` actually makes little sense here?

Reply:
I think we should drop that and leave it as shown above. So anything after the second `=` becomes arguments. WDYT @JacobValdemar ?

Reply:
It definitely makes more sense without `params=`. But I also think it's quite odd to use '=' as a separator in the first place since it's very often used as the opposite like `env=dev,strict=true,host=5432` (where `,` obviously is the separator). Or is = separator normal in this context?

Reply:
I don't think there is precedence. It is not connection string arguments all of it. But maybe it should be?

Currently multiple hosts are defined like this:

```
postgresql-controller --host-credentials=host1:1234=user1:pass1,host2:4321=user2:pass2
```

As multiple hosts are defined with `,` as separator, that would clash with a normal connection string.

Example here: https://github.com/lunarway/k8s-cluster-config/blob/master/dev/releases/operators/postgresql-controller/40-deployment.yaml#L65

Can we come up with something completely different in terms of specifying this? A complete connection string instead maybe?
This is what they do in some of the launchpad services I can see: https://sourcegraph.lunar.tech/github.com/lunarway/lunar-way-bank-account-service/-/blob/shuttle.yaml#L102

Reply:
It does require more from us to parse the host name from that, but that might be OK? It would make it much more recognizable?

Reply:
I approve if I think it would work and it's not obviously a mess, but maybe I should be more strict? What makes code ready for approval? Should I have to understand everything and it to be perfect? 🤔

Reply:
If it is hard to understand for you, it properly is for others as well 😉 In that case you can choose to just comment instead of approving/requesting changes. :)

Reply:
Idk how many places this is used. Maybe we should just stick with removing params=, to not make too many breaking changes, and let = be used as a separator. After all this is only a one hour task?

Reply:
Good point 👍 I'll drop the prefix for now. 💯